### PR TITLE
SAK-47489 this disables SSE for Bullhorns and modifies the fetch to the /direct/ endpoint

### PR DESCRIPTION
**Pull request by ottenhoff**
SSE is awesome as it reduces the cost of polling, and we should move more tools to use it. We also need more experience with SSE and managing connections.

This commit:

a) Sets up the number of alerts in portal server page load
b) Avoids a fetch call to setup bullhorns
c) Bullhorns only called onclick
d) Disables setting up the SSE

To use SSE, I'd like to see one more tool use it (not just Bullhorns) and would like to see more research into how to resume an SSE conn.

### CODE REVIEW

File: library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js [hunk 1]
```diff
@@ -217,20 +217,30 @@
...
-        text: function (event, api) {
+      events: {
+        visible: function (event, api) {
 
-          if (portal.bullhornAlerts && portal.bullhornAlerts.length <= 0) {
-            return portal.wrapNoAlertsString(portal.bullhornsI18n.noAlerts);
+          var firstBunch = document.querySelector("#bullhorn-alerts button");
+          if (firstBunch) {
+            firstBunch.focus();
+          }
```
Comment (adrianfish) [inline library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js:226]:
```suggestion
          const firstBunch = document.querySelector("#bullhorn-alerts button");
          firstBunch && firstBunch.focus();
```

File: library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js [hunk 2]
```diff
@@ -273,7 +271,7 @@
     horn.append('<span id="bullhorn-counter" class="bullhorn-counter-red">' + count + '</span>');
   };
 
-  var updateCount = function (count) {
+  let updateBullhornCounter = function (count) {
```
Comment (adrianfish) [inline library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js:274]:
```suggestion
  const updateBullhornCounter = function (count) {
```

File: portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeStandardHead.vm [hunk 3]
```diff
@@ -7,15 +7,11 @@
...
-        <script> ## Include this at the top so tool markup and headscripts.js can use it
-            ## SAK-16484 Allow Javascript to easily get at user details.
-            ## SAK-13987, SAK-16162, SAK-19132 - Portal Logout Timer
+        <script>
             var portal = {
                 "bullhorns": {
                     "enabled": $!{useBullhornAlerts},
-                    #if ($useBullhornAlerts)
-                    "pollInterval": $!{bullhornsPollInterval}
-                    #end
```
Comment (adrianfish) [inline portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeStandardHead.vm:18]: Spiffing. Well spotted!

**Review by adrianfish — APPROVED** [review]
Some small stuff

**Comment by SedueRey** [comment]
Hi, @ottenhoff , should we also disable it on backend? I don't know if anyone with the URL could run it on local (with an infinite loop on devtools, for example)

Thanks!!

**Comment by ottenhoff** [comment]
Sure, you can block /api/users/me/events if you want to be sure.

**Pull request merged**